src/xdocs/google_style.xml

```diff
@@ -1485,6 +1501,35 @@
                     test</a>
                 </td>
               </tr>
+              <tr>
+                <td><a name="4.8.5.5"/>
+                  <a href="#4.8.5.5">
+                    <span class="wrapper inline">
+                      <img src="images/anchor.png" alt=""/>
+                    </span>
+                  </a>
+                </td>
+                <td>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s4.8.5.5-local-parameter-annotation-style  ">
+                    4.8.5.5 Parameter and local variable annotations
+                  </a>
+                </td>
+                <td>
+                  <span class="wrapper inline">
+                    <img
+                        src="images/ban_red.png"
+                        alt="" />
+                  </span>
+                  Type-use annotations cannot be detected by Checkstyle due to it's
+                  <a href="https://checkstyle.org/writingchecks.html#Limitations">
+                    limitations</a>,
+                  see issue
+                  <a href="https://github.com/checkstyle/checkstyle/issues/15416">
+                    #15416</a>
+                  for more detailed explanation.
+                </td>
```

Comment:
There were particularly no rule for [this section](https://google.github.io/styleguide/javaguide.html#s4.8.5.5-local-parameter-annotation-style) expect when type-use annotation is used:

> There are no specific rules for formatting annotations on parameters or local variables (except, of course, when the annotation is a type-use annotation).

To show that we dont support type-use annotation, I copied the message from 4.8.5.1 Type-use annotations:

https://github.com/checkstyle/checkstyle/blob/2d3546412b0a0e142455d77f3bf3bcc26d5d7032/src/xdocs/google_style.xml#L1373-L1390